docs/_documentation/contributing/contribute.md

```diff
@@ -31,7 +31,9 @@ The current state of Visual Studio for Mac doesn't allow us to build MvvmCross.
 
 On Windows, simply run MvvmCross in Visual Studio. Note: you need Visual Studio 2017 15.7 or higher to be able to compile.
 
+To debug into the source code of MvvvCross on Mac with Visual Studio for Mac, run `msbuild MvvmCross.sln /t:Build /p:Configuration=Debug` first to generate the local dlls, then create VSMac projects and add the generated dlls as local .Net Assembly references. Both break point and line-by-line debugging are supported.
```

Comment:
MvvvCross -> MvvmCross
.Net -> .NET
Visual Studio for Mac -> Visual Studio for Mac (VSM)
VSMac -> VSM
Break point -> break points
dlls -> DLLs

Reply:
Thanks for the review, I have updated the PR according to your suggestions.